app/src/main/java/com/github/se/wanderpals/model/viewmodel/ExpenseViewModel.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.se.wanderpals.model.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import com.github.se.wanderpals.model.data.Expense
+import com.github.se.wanderpals.model.data.User
+import com.github.se.wanderpals.model.repository.TripsRepository
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+/**
+ * ViewModel for the Expense
+ *
+ * @param tripsRepository Repository to fetch data from
+ * @param tripId Id of the trip
+ */
+open class ExpenseViewModel(private val tripsRepository: TripsRepository, tripId: String) :
+    ViewModel() {
+
+  private val _users = MutableStateFlow(emptyList<User>())
+  open val users: StateFlow<List<User>> = _users.asStateFlow()
+
+  init {
+    loadMembers(tripId)
```

Comment:
Line 28-29 were probably used to do tests before linking to the database. So theses lines can be removed.